.editorconfig

```diff
@@ -0,0 +1,81 @@
+# .editorconfig for the Exercism AWK track repository
+# https://spec.editorconfig.org/
+
+# Top-most EditorConfig file
+# A best practice to prevent settings from parent directories from interfering.
+root = true
```

Comment:
This would prevent people from having their own overrides? Why would we prevent people from having additional settings?

Reply:
This `root = true` works exclusively for awk repo. If I have my own `.editorconfig` in home, and open some different folder, my settings will be used.